apps/rush-lib/src/data/ChangeManagement.ts

```diff
@@ -80,3 +87,87 @@ export interface IChangeInfo {
    */
   type?: string;
 }
+
+/**
+ * Writes the change file to disk in sync mode.
+ * @internal
+ *
+ * @param rushConfiguration - Rush configuration
+ * @param changeFile - change file data
+ */
+export function _writeChangeFileSync(
```

Comment:
I don't understand the purpose of this function, especially since it is not being used by ChangeAction.ts

---
In reply to: [126511714](https://github.com/Microsoft/web-build-tools/pull/267#discussion_r126511714) [](ancestors = 126511714)